### ISSUE TITLE
Use new base image for Dockerfile as CentOS 8 is EOL

### ISSUE THREAD

**Issue by matthewfeickert** [yadage/packtivity#83]
The current builds of the Dockerfile fail at

```
Step 2/6 : RUN dnf install -y python3
 ---> Running in fc62e180257f
CentOS Linux 8 - AppStream                      [26](https://github.com/yadage/packtivity/runs/5047398152?check_suite_focus=true#step:4:26)5  B/s |  38  B     00:00    
Error: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist
The command '/bin/sh -c dnf install -y python3' returned a non-zero code: 1
```

which [is happening because](https://forums.centos.org/viewtopic.php?f=54&t=78708)

> CentOS 8 went EOL at the end of December [2021] and in line with all the public announcements, the content of the CentOS 8 repos has been moved to vault.centos.org.

So that means that all CentOS 8 Dockerfiles are now broken forever. :cry: 

We need to choose a new base image for `packtivity`'s Dockerfile, so do we go to CentOS 7, switch to Debian, or go to Fedora?

**Comment by matthewfeickert** [comment]
@lukasheinrich Before you used

https://github.com/yadage/packtivity/blob/e7020f549bb0933afc1f63c399ee0926d113c23c/Dockerfile#L1

for the image you were using

https://github.com/yadage/packtivity/blob/a46590ad2df6fd96e8660c32b609a11ebf70a5b3/Dockerfile#L1

(but that comes from the first commit back in March 11, 2016).

Thoughts on a future base image?

**Comment by matthewfeickert** [comment]
Resolved by PR #85.
